### PR TITLE
Added frontend challenges websites

### PR DESCRIPTION
**Pull request by naiiiden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -991,7 +991,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
 - :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
 - :video_camera: [Frontend with HTML CSS JS & Browser APIs](https://youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC)
-
+- :other: [Frontend Coding Challenges](https://www.frontendmentor.io)
```
Comment (shahednasser) [inline README.md:994]: Please change the title to the page's title

File: README.md [hunk 2]
```diff
@@ -991,7 +991,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
...
+- :other: [Frontend Coding Challenges](https://www.codewell.cc)
```
Comment (shahednasser) [inline README.md:995]: Please change the title to the page's title.

Also, please add a new empty line after this.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @naiiiden for your contribution. I've added some comments below.

File: README.md [hunk 3]
```diff
@@ -991,8 +991,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
 - :video_camera: [Frontend Interview Series](https://youtube.com/playlist?list=PL-Jc9J83PIiFXzRwSTSwqL7U2hoMk6K6F)
 - :video_camera: [Frontend with HTML CSS JS & Browser APIs](https://youtube.com/playlist?list=PL-Jc9J83PIiHU9RkY9sfh3G64-bd0ptvC)
-- 💡 [Design To Code Frontend Challenges](https://www.frontendmentor.io)
-- 💡 [Design To Code Frontend Challenges](https://www.codewell.cc)
+- :bulb: [Frontendmentor.io Design To Code Frontend Challenges](https://www.frontendmentor.io)
+- :bulb: [Codeell.cc Design To Code Frontend Challenges](https://www.codewell.cc)
```
Comment (shahednasser) [inline README.md]: Can you fix the typo in the name here?

Comment (naiiiden) [reply README.md]: yes, check commit "fixed typo"

**Comment by shahednasser** [comment]
Great thanks!

**Pull request merged**